internal/importers/base/importer_test.go

```diff
@@ -400,6 +419,9 @@ func TestBaseImporter_ImportProjects(t *testing.T) {
 		name := name
 		tc := tc
 		t.Run(name, func(t *testing.T) {
+			if parallelAllowed() {
```

Comment:
All of the importer tests use `internal/importers/importertest/testcase` which handles making a test helper. When we turn parallel tests back on, I believe  change should be made there, and not here.

https://github.com/golang/dep/blob/bf8f29518fd7074cf051621741d442bfc9971076/internal/importers/importertest/testcase.go#L48-L53